### DISCUSSION TITLE
Which BTs present in Contract award notices can be modified and published in a Contract modification notice?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#246] [Q&A] [answered]
Hi,
Can you point me to some documentation where I can identify which BTs in CANs that are connected to contract information and can be modified and presented in a Contract modification notice?

Kind regards,
Åsa

**Comment by rouschr** [comment]
The SDK fields.json "forbidden" and "noticeTypes" is the closest thing to answer this questions (technically speaking):
https://github.com/OP-TED/eForms-SDK/blob/develop/fields/fields.json

For a non-technical "business" answer or more docs oriented answer, maybe @YvesJo can help you out.

**Comment by YvesJo** [comment, marked as answer]
For business related questions, please contact the helpdesk ted@publications.europa.eu
